test/integration/verify/controls/aws_transit_gateway.rb

```diff
@@ -0,0 +1,14 @@
+title 'Test AWS Transit Gateway'
+
+aws_kms_key_enabled_id = attribute(:aws_transit_gateway_id, default: '', description: 'The AWS Transit Gateway ID.')
```

Comment:
This looks like a copy-paste error. Perhaps this could be updated to `aws_transit_gateway_id `

Reply:
good lord you are right, will fix that